person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def __is_of_age?
+    return unless @age >= 18
+  end
+
+  def can_use_services?
+    if __is_of_age? || @parent_permission
```

Comment:
- [ ] I know that you have a rubocop error with the name of the method `is_of_age?`, but it's not a good practice to name it as you did, so please name it `of_age?` instead.

- [ ] You used unless the wrong way here, Please make the `of_age?` method like the following to make it function as expected:
```ruby
 def of_age?
    @age >= 18
  end
```